requirements.txt

```diff
@@ -27,3 +27,4 @@ pandas
 # extras --------------------------------------
 thop  # FLOPS computation
 pycocotools>=2.0  # COCO mAP
+sparseml>=0.2
```

Comment:
we may want to change this to ~ for now because we have some changes coming in that will be breaking